extensions/applicationinsights-analytics-js/Tests/Unit/src/AnalyticsExtensionSize.tests.ts

```diff
@@ -43,7 +43,7 @@ export class AnalyticsExtensionSizeCheck extends AITestClass {
                         return;
                     } else {
                         return response.text().then(text => {
-                            let size = Math.ceil(pako.deflate(text).length/1024);
+                            let size = Math.ceil((pako.deflate(text).length/1024) * 100) / 100.0;
```

Comment:
Is this used to change the format of the number? weird to multiply and divide by the same number

Reply:
Yes this is for formatting the output number only so we get it at most for 2 decimal places.